web/src/components/common/DeadlineTag.tsx

```diff
@@ -42,11 +48,11 @@ interface DeadlineTagProps {
  * is not over.
  * Displays number of days since deadline if listing has ended.
  */
-const DeadlineTag: React.FC<DeadlineTagProps> = ({deadline}) => {
+const DeadlineTag: React.FC<DeadlineTagProps> = ({deadline, largerFont}) => {
```

Comment:
can you explain what is largerFont for in the comment? same for other files

Reply:
@ifhuang  Oh it just makes the component use a larger font for the text when the prop is passed in